Modules/Services/Sources/Models/SubscriptionData/SubscriptionId.swift

```diff
@@ -0,0 +1,7 @@
+public struct SubscriptionId: Hashable {
```

Comment:
Do we really need this structure? Why not just use string or typealias?

Reply:
I think we can delete it

Reply:
SubscriptionId was deleted